docs/example/api.html

```diff
@@ -406,6 +407,34 @@ <h3 id="functions_listVoters">listVoters</h3>
                     </div>
                 </div>
 
+                <h3 id="functions_broadcastSignedTransaction">broadcastSignedTransaction</h3>
+
+                <div class="panel panel-default">
+                    <div class="panel-body">
+                        <div class="rows">
+                            <div class="col-md-12">
+                                <h3>Example <small>Create a signed transaction, use broadcastSignedTransaction to send it to the network</small></h3>
+                                <figure class="highlight">
+                                    <pre>
+                                        <code class="language-html" data-lang="html">
+var LSK = lisk.api({testnet: true});
+
```

Comment:
Yes, the html formating within <code> tags looks weird. But that is necessary for the display.